medical_pathology/demo/medical_pathology_category_demo.xml

```diff
@@ -0,0 +1,57 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2016 LasLabs Inc.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl). -->
+
+<odoo>
+
+    <record id="medical_pathology_category_1" model="medical.pathology.category">
```

Comment:
I'm going to hold off on this one - it causes cascading issues in our incubator & I've already resolved them once this version. We'll hold off till next version.